custom/ewsghana/tasks.py

```diff
@@ -189,6 +190,13 @@ def migrate_email_settings(domain):
     endpoint = GhanaEndpoint.from_config(config)
     migrate_email = EmailSettingsSync(domain)
 
+    for user in iter_docs(WebUser.get_db(), WebUser.ids_by_domain(domain)):
+        for notification in ReportNotification.by_domain_and_owner(domain, user['_id'], stale=False):
+            notification.delete()
```

Comment:
Does this mean that you're going to delete all emailed reports and then resync them from the source? I don't think that's the best approach since people could have created emailed reports in the meantime that don't exist in the source.

Can we instead just have a one-off task that deletes all emailed reports that use an archived location as the location filter?